wpilib/_impl/start.py

```diff
@@ -32,6 +36,8 @@ def _start():
                 finally:
                     hal.exitMain()
 
+            signal.signal(signal.SIGINT, self._on_ctrl_c)
```

Comment:
Wouldn't it be better to wrap the runMain in a try/except?

Reply:
I don't see why it matters in this case.